src/ultra/keys/UltraVerificationKey.sol

```diff
@@ -4,7 +4,7 @@
 pragma solidity >=0.8.4;
 
 library UltraVerificationKey {
-    function verificationKeyHash() internal pure returns(bytes32) {
+    function verificationKeyHash() internal pure returns (bytes32) {
```

Comment:
we should make a pr into barretenberg to fix this formatting

Reply:
Think there is a forge setting that we might just be missing that removes it, should give us the same as in barretenberg then.